server/routes/appointments/appointmentsController.ts

```diff
@@ -878,14 +878,14 @@ export default class AppointmentsController {
     )
   }
 
-  private async getCaseWorker(accessToken: string, username: string): Promise<AuthUserDetails | string> {
+  private async getCaseWorker(accessToken: string, userId: string): Promise<AuthUserDetails | null> {
```

Comment:
i simplified this method, so it doesn't return the username on 404. the caller handles it instead.